pkg/arvo/app/graph-pull-hook.hoon

```diff
@@ -9,7 +9,7 @@
       update:store
       %graph-update
       %graph-push-hook
-      1  1
+      2  2
```

Comment:
Pull-hook could probably be backwards compatible with old publishers. Doesn't have to be though.